plow/backends/rename_ids.flow

```diff
@@ -1,38 +1,49 @@
+import ds/treeutils;
 import plow/backends/bprogram;
 import plow/backends/pretty;
 import plow/dexp/pretty;
 
 export {
+	// This struct stores the information about original names. Used for reflection in backends.
```

Comment:
Is the original name the key and the new the value? If so, state that.

Reply:
fixed